src/components/BulkEnrollmentModal/index.jsx

```diff
@@ -101,18 +111,17 @@ class BulkEnrollmentModal extends React.Component {
         ref={this.errorMessageRef}
         tabIndex="-1"
       >
-        <Alert
-          variant="danger"
-          dismissible
-        >
+        <Alert variant="danger">
           <Alert.Heading>
             Before you can invite learners to a course they must have a subscription license.
           </Alert.Heading>
-          <p className="alert-body">
+          <p className="alert-body text-justify">
             The following emails addresses are not yet associated with a subscription.
             Please take note of the following learners and invite them from the subscription management page.
           </p>
-          {failedLearners.map((failedLearner) => <p className="alert-email" key={`${failedLearner}-email`}>{failedLearner}</p>)}
+          {failedLearners.map((failedLearner) => (
+            <p className="mb-1 alert-email" key={`${failedLearner}-email`}>{failedLearner}</p>
```

Comment:
If this is a list, it **must** be a list, not a seres of `p` tags, to be a11y compliant.
Please change to a series of `li` tags wrapped by a `ul`